javaagent-tooling/src/test/java/io/opentelemetry/javaagent/tooling/bytebuddy/matcher/ExtendsClassMatcherTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.tooling.bytebuddy.matcher;
+
+import static io.opentelemetry.javaagent.extension.matcher.AgentElementMatchers.extendsClass;
+import static net.bytebuddy.matcher.ElementMatchers.named;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.opentelemetry.javaagent.tooling.muzzle.AgentTooling;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.pool.TypePool;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.objectweb.asm.Opcodes;
+
+class ExtendsClassMatcherTest {
+
+  private static TypePool typePool;
+
+  @BeforeAll
+  static void setUp() {
+    typePool =
+        AgentTooling.poolStrategy()
+            .typePool(
+                AgentTooling.locationStrategy()
+                    .classFileLocator(ExtendsClassMatcherTest.class.getClassLoader(), null),
+                ExtendsClassMatcherTest.class.getClassLoader());
+  }
+
+  @ParameterizedTest
+  @CsvSource({
+    "io.opentelemetry.javaagent.tooling.bytebuddy.matcher.testclasses.A, io.opentelemetry.javaagent.tooling.bytebuddy.matcher.testclasses.B, false",
```

Comment:
could consider dropping the package name from here and adding it inside the test method

Reply:
that makes it just harder to find usages in the IDE